knowledge_base/wiki/ERD.md

```diff
@@ -0,0 +1,503 @@
+# Autogenerated Mermaid ERD
```

Comment:
I think a slightly. more descriptive file name would help contextualize what this document is and where it's headed. Maybe even just "Database ERD"?